ivy/functional/backends/paddle/elementwise.py

```diff
@@ -976,6 +1040,11 @@ def log(x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None) -> paddle.T
 def exp(x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None) -> paddle.Tensor:
     if x.dtype in [paddle.int32, paddle.int64, paddle.float32, paddle.float64]:
         return paddle.exp(x)
+    if x.dtype in [paddle.complex64, paddle.complex128]:
+        return paddle.multiply(
+            paddle.exp(x.real()),
+            paddle.complex(paddle.cos(x.imag()), paddle.sin(x.imag())),
+        )
     return pow(math.e, x).astype(x.dtype)
```

Comment:
I guess while we are at it
```suggestion
    return paddle_backend.pow(math.e, x).astype(x.dtype)
```

Reply:
I don't see why this is needed, since the case of complex `x` is already handled (and handled more efficiently than using `paddle_backend.pow(math.e, x)` since we use the special case of the base being real)

Reply:
this change isn't concerned with complex input, `pow` here refers to the paddle backend implementation of `pow` but if we use `pow` instead of `paddle_backend.pow` we are actually calling `ivy.pow` and hence passing through all the wrappers that we don't need

Reply:
I thought since `pow` is defined in this file, `pow` and `paddle_backend.pow` are the same function? Done either way, since its more clear with `paddle_backed.` if nothing else